controllers/users-controllers.js

```diff
@@ -0,0 +1,9 @@
+const {readUsers} = require("../models/users-models")
+
+
+
+exports.getAllUsers = (req, res, next) =>{
+    readUsers().then((users) =>{
+        res.status(200).send(users)
```

Comment:
Could you just amend this slightly so it's coming back on a key of users please?